src/components/List/utils.tsx

```diff
@@ -1,11 +1,5 @@
 import React from 'react';
-import Highlighter from 'react-highlight-words';
-
-import { listLabel, listLabelHelperText, listLabelWithHelper } from './List.style';
-import { ListItemType } from './types';
-import { FilterOption } from '../Filter';
-import Icon from '../Icon';
-import { SelectOption } from '../Select';
+// import Highlighter from 'react-highlight-words';
```

Comment:
comment

Reply:
This is now responsibility of `Select` and the same as https://github.com/Orfium/orfium-ictinus/pull/655#discussion_r1279028881

leaving TODO for now at [`3b16bc4` (#655)](https://github.com/Orfium/orfium-ictinus/pull/655/commits/a660901b79c69290ad80ea9d7583c097fac80daa)